src/applications/ivc-champva/10-10D/components/Sponsor/sponsorFileUploads.js

```diff
@@ -117,7 +117,7 @@ export function acceptableFileList(list) {
 export const requiredFileUploadMessage = {
   'ui:description': (
     <p>
-      <i>This file is required for your application.</i>
+      <em>This file is required for your application.</em>
```

Comment:
Linter was flagging this as an icon, so switched to `<em>` which has the same effect of italicizing the words but without the false-positive icon flag